internal/winapi/offlinereg.go

```diff
@@ -0,0 +1,36 @@
+package winapi
+
+// Offline registry management API
+
+type OrHKey uintptr
+
+type RegType uint32
+
+const (
+	// Registry value types: https://docs.microsoft.com/en-us/windows/win32/sysinfo/registry-value-types
+	REG_TYPE_NONE                       RegType = 0
+	REG_TYPE_SZ                         RegType = 1
+	REG_TYPE_EXPAND_SZ                  RegType = 2
+	REG_TYPE_BINARY                     RegType = 3
+	REG_TYPE_DWORD                      RegType = 4
+	REG_TYPE_DWORD_LITTLE_ENDIAN        RegType = 4
+	REG_TYPE_DWORD_BIG_ENDIAN           RegType = 5
+	REG_TYPE_LINK                       RegType = 6
+	REG_TYPE_MULTI_SZ                   RegType = 7
+	REG_TYPE_RESOURCE_LIST              RegType = 8
+	REG_TYPE_FULL_RESOURCE_DESCRIPTOR   RegType = 9
+	REG_TYPE_RESOURCE_REQUIREMENTS_LIST RegType = 10
+	REG_TYPE_QWORD                      RegType = 11
+	REG_TYPE_QWORD_LITTLE_ENDIAN        RegType = 11
+)
+
+//sys OrMergeHives(hiveHandles []OrHKey, result *OrHKey) (win32err error) = offreg.ORMergeHives
```

Comment:
nit but why did you choose to lower case the r in `Or` instead of matching the case from the API?

Reply:
Thanks for asking this. Go guidelines suggest that we use "OR" instead of "Or" anyway. But while making this change I also realized that sometime back a change was already made to the main branch that added some of the same offline registry APIs that this PR adds. I have now handled that too.